sigle/src/modules/publicHome/components/PublicHome.tsx

```diff
@@ -232,32 +266,109 @@ export const PublicHome = ({ file, settings, userInfo }: PublicHomeProps) => {
                   : `@${twitterHandle}`}
               </Typography>
             )}
+            <Box
+              css={{ width: '1px', height: '$4', backgroundColor: '$gray9' }}
+            />
+            <Flex
+              align="center"
+              gap="2"
+              css={{
+                '&:hover': {
+                  '& button': {
+                    display: 'block',
+                  },
+                },
+              }}
+            >
+              <Typography
+                size="subheading"
+                css={{
```

Comment:
as this css is being shared among all the items in this row, maybe it make sense to create a styled component at the beginning of this file?